core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1748,4 +1749,10 @@ CacheConfigurationException offHeapMemoryEvictionSizeNotLargeEnoughForAddresses(
 
    @Message(value = "Cannot acquire lock '%s' for persistent global state", id = 512)
    CacheConfigurationException globalStateCannotAcquireLockFile(@Cause Throwable cause, File lockFile);
+
+   @Message(value = "Memory based eviction requires a transactional cache that doesn't allow for 1 phase commit or synchronizations", id = 513)
```

Comment:
Should it be "Exception based eviction requires (...)" ?

Reply:
Yes, thanks.